_includes/official_resources.md

```diff
@@ -1,9 +0,0 @@
-Athough AEON is community driven, official resources are considered to be those maintained by Aeon's initial developer, Smooth. Additional support is provided by Stoffu as well as many community contributors. Remember that anyone can contribute to the Aeon project.
-
```

Comment:
I am really not in agreement about this statement

```
Athough AEON is community driven, official resources are considered to be those maintained by Aeon's initial developer, Smooth. Additional support is provided by Stoffu as well as many community contributors. Remember that anyone can contribute to the Aeon project.
```

Basically what this says to me : Aeon has a community but it is owned by smooth. I thought Aeon was decentralized but now you say it is run by a single person?? All my aeon is in their hands?? Very confusing and incorrect and I don't think even smooth would like this. .

Also 
`Remember that anyone can contribute to the Aeon project.`

Any one **can** is not very a good way to put it. You should say rather that every one is invited as it is more welcoming.